modules/credential/src/main/java/org/eclipse/xpanse/modules/credential/Credential.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.credential;
+
+import org.eclipse.xpanse.modules.credential.enums.CredentialType;
+
+/**
+ * The credential object.
+ */
+public class Credential {

```

Comment:
Would it make sense to name it as 'CredentialVarible'?

Reply:
I think it is not suitable to name it `CredentialVariable`, 
because we are not sure about all kinds of credentials are Variables. variables should only be one of kind of them.